File: src/SegitigaForloop.java
public class SegitigaForloop {
    public static void main(String[] args) {
        triangle(5); // tentuin jumlah baris segitiganya
    }

    private static void triangle(int x) {
        // space = jumlah spasi (tiap baris harus ngurang 1)
        int space = x -1;
        // for loop untuk jumlah baris
        for (int i = 0; i < x; i++) {
            // for loop untuk mencetak spasi
            for (int j = 0; j < space; j++) {
                System.out.print(" ");
            }
            space -= 1;

            // for loop untuk mencetak *
            for (int j = 0; j < i + 1; j++) {
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}

File: src/segitiga_forloop.py
def triangle(x):
    # k = jumlah spasi (tiap baris harus ngurang 1)
    k = x - 1
    # for loop untuk jumlah baris
    for i in range(0, x):
        # for loop untuk mencetak spasi
        for j in range(0, k):
            print(end=" ")
        k = k - 1

        # for loop untuk mencetak *
        for j in range(0, i + 1):
            print("* ", end="")
        # \r biar rapi
        print("\r")


# tentuin jumlah baris segitiganya
n = 5
triangle(n)
